azure-pipelines.yml

```diff
@@ -1,5 +1,14 @@
 trigger:
-- master
+  batch: true
+  branches:
+    include:
+    - master
+
+pr:
```

Comment:
Added this to avoid overriding settings in the AzureDev Ops GUI. This will make the pipeline trigger when there is a PR on master or v4x

Reply:
If we include this pipeline for PRs, don't we run the risk of accidentally shipping builds from PRs (like I did recently on `ms-rest-nodeauth`)?

Reply:
If we set up auto publish yes. I've disabled auto publish for this package. But this is a very good point, I think we may want to have separate pipelines for CI (merge) and PR and just have CI plugged in to the auto release pipeline